server/store.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"github.com/go-redis/redis/v8"
+)
+
+type Store interface {
+	Save(key string, data string, expires time.Duration) error
+	Load(key string) (string, error)
+	Delete(key string) error
+}
+
+type MapSyncStore struct {
+	StoreData sync.Map
```

Comment:
I don't think we need `StoreData` to be public. It should be only used internally by `MapSyncStore` objects

Reply:
And also, rename that to something more meaningful, such as `m`